### ISSUE TITLE
if-then-else example does not work

### ISSUE THREAD

**Issue by zbuc** [jqlang/jq#1569]
The example for if-then-else in the documentation does not work correctly when you try to run it in the linked jqplay editor:

https://stedolan.github.io/jq/manual/#example74

https://jqplay.org/jq?q=%27if%20.%20%3D%3D%200%20then%0A%20%20%22zero%22%0Aelif%20.%20%3D%3D%201%20then%0A%20%20%22one%22%0Aelse%0A%20%20%22many%22%0Aend%27&j=2

Results:

```
jq: error: syntax error, unexpected INVALID_CHARACTER, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
'if . == 0 then
jq: 1 compile error
exit status 3
```

**Comment by pkoppstein** [comment]
@zbuc - When using jqplay, you don't include the outermost quotation marks.  Same thing if the jq program goes in a file.  Indeed, the outermost quotation marks are sometimes not needed at all.

**Comment by zbuc** [comment]
I followed the link directly from the documentation page, and did not manually change anything. It does not work as linked from the documentation.

**Comment by nicowilliams** [comment]
@zbuc That's an example for you to use on a local shell, not on jqplay.  I'm sorry, but there's no way to make the examples work for both.

**Comment by zbuc** [comment]
Then why is there a link to run it on jqplay if it doesn't work?

**Comment by zbuc** [comment]
@nicowilliams here, I found and fixed the bug. The regex being used only worked properly on single-line examples and failed on multiline ones.

https://github.com/stedolan/jq/pull/1570

**Comment by nicowilliams** [comment]
@zbuc Oh!  I never knew about that feature (the Run button).  I use NoScript, and wasn't enabling the required origin for that...

**Comment by itchyny** [comment]
This has been resolved by #1716, and also the commit 048acee39babe759d1db0f32f263857c27361746 in #1825 removes the newlines from the example, and the page was rebuilt at 76c044f301c3d33b122974e405a80f2b0775dce7, the issue no longer happens.
